sky/clouds/oci.py

```diff
@@ -296,19 +298,36 @@ def _make(instance_list):
     def check_credentials(cls) -> Tuple[bool, Optional[str]]:
         """Checks if the user has access credentials to this cloud."""
 
+        credential_help_str = (
+            'To configure credentials, go to: '
+            'https://docs.oracle.com/en-us/iaas/Content/API/Concepts/'
+            'apisigningkey.htm\n'
+            f'{cls._INDENT_PREFIX}Please make sure the API keys and the config '
+            'files are placed under ~/.oci:\n'
+            f'{cls._INDENT_PREFIX}  ~/.oci/config\n'
+            f'{cls._INDENT_PREFIX}  ~/.oci/oci_api_key.pem\n'
+            f'{cls._INDENT_PREFIX}The ~/.oci/config file should have the '
+            'following format:\n'
+            f'{cls._INDENT_PREFIX}  [DEFAULT]\n'
+            f'{cls._INDENT_PREFIX}  user=ocid1.user.oc1..aaaaaaaa\n'
+            f'{cls._INDENT_PREFIX}  '
+            'fingerprint=aa:bb:cc:dd:ee:ff:gg:hh:ii:jj:kk:ll:mm:nn:oo:pp\n'
+            f'{cls._INDENT_PREFIX}  tenancy=ocid1.tenancy.oc1..aaaaaaaa\n'
+            f'{cls._INDENT_PREFIX}  region=us-sanjose-1\n'
+            f'{cls._INDENT_PREFIX}  key_file=~/.oci/oci_api_key.pem')
+
         try:
             # pylint: disable=import-outside-toplevel,unused-import
             import oci
         except ImportError:
-            return False, ('oci is not installed. Install it with: '
-                           'pip install oci')
+            return False, ('`oci` is not installed. Install it with: '
+                           'pip install oci\n'
+                           f'{cls._INDENT_PREFIX}{credential_help_str}')
```

Comment:
This becomes quite a long message (compared to other clouds) and since a large set of users will not have OCI, the output for `sky check` will be an unnecessarily long whenever they run it.
<img width="987" alt="image" src="https://github.com/skypilot-org/skypilot/assets/4416605/29ee445a-31cb-4281-a1c7-e877526b7f2d">

Should we change the message here to `oci is not installed. Install it with: 'pip install oci'\n For more details, refer to <OCI installation link>`.

L330 is okay to keep since it triggers only if a user has oci installed but not the credentials setup.

Reply:
Good idea! Changed to the link.